package.json

```diff
@@ -75,6 +75,7 @@
         "@semantic-release/git": "^9.0.0",
         "@types/chai": "^4.2.3",
         "@types/mocha": "^9.0.0",
+        "@types/mparticle__web-sdk": "^2.16.0",
```

Comment:
Note to self: Revert package-lock.json so that only changes related to type definitions are included.